### ISSUE TITLE
Missing Copy/Save buttons

### ISSUE THREAD

**Issue by curi0usJack** [flameshot-org/flameshot#2399]
### Flameshot Version

11.0.0-4

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Arch 5.16.7

### Description

Since upgrading to 11, the Copy and Save buttons are now missing when taking a screenshot. I see the checkmark and x buttons, and can double click to copy _and_ save, but I'd prefer the UI options to choose either/or.

### Steps to reproduce

1. Ctrl+Shift+C to start a screenshot
2. Drag an area
3. Note the missing buttons.

### System Information

i3/xorg

**Comment by mmahmoudian** [comment]
Let me guess, in your i3 you have bounded your keyboard shortcut to `flameshot gui -p`. Right? If so, it means you have missed reading the release notes, you have not searched in the existing issues to see if anyone have reported it, and you didn't checkout the pinned issues!

Read this link for more info:

https://flameshot.org/docs/advanced/commandline-options/#final-actions

**Comment by mmahmoudian** [comment]
@curi0usJack 
I disagree, being a dick would have been to close this issue, mark it as a duplicate and just post a link. Instead I wrote a polite curse-free text, explained the possible cause and my hunch, pointed you to a correct direction, marked the issue as "waiting for info" and waited to see if your issue is resolved. In return what did I get? I was called a dick by you!

Nonetheless, I apologize for the unfriendly tone of my previous message.

It worth to reiterate that every single day we have multiple users filing bug report and claim that the keyboard shortcut is not working, despite the fact that:
- there is a release note
- there is a dedicated part on our website
- there are many many other similar false alarms on the issue
- there is a pinned issue (screenshot below) 

When you filled the form, you were kindly and politely asked to do a little bit of research before filing an issue (screenshot below).  I am nice and I'm trying to stay nice, but everything in life has a limit! 

![image](https://user-images.githubusercontent.com/390889/153435708-ab83eb7a-0a27-42b2-ab1e-b519f8e49c89.png)
![image](https://user-images.githubusercontent.com/390889/153436028-e3498fce-b7e4-4dda-af8a-01e891678196.png)

-------


>  I also searched existing and closed issues for keywords that I thought appropriate.

With all due respect, then let's see what I can find using the words in the title that you yourself chose for your issue.

![image](https://user-images.githubusercontent.com/390889/153437264-54d40139-68e8-465e-99c9-05c195b5ae5b.png)


Also check how many posts are in each of those issues. On average it takes 17.33333 posts (and counting) to "solve" each of those, so one person not putting minimal effort in doing a simple search would takes lots and lots of time and energy from everyone. Don't you agree?

All these said, I again apologize for the tone.

Have a good day
